### PR TITLE
Update to openstudio 3.8, e+ 24.1 and pyfmi 2.11

### PR DESCRIPTION
**Pull request by TShapinsky**

### CODE REVIEW

**Comment by TShapinsky** [comment]
@anyaelena @kbenne This should be pretty much good. Everything works now. Once the official release of 3.8 is released we can swap that in with this and then merge it into develop

**Comment by TShapinsky** [comment]
@anyaelena @kbenne This branch is up to date with the release of OpenStudio 3.8. When we're ready I can do a new release of the alfalfa dependencies and then merge this in here.

File: alfalfa_worker/jobs/openstudio/lib/workflow/measures/alfalfa_prepare_idf/measure.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # *******************************************************************************
-# OpenStudio(R), Copyright (c) 2008-2022, Alliance for Sustainable Energy, LLC.
+# OpenStudio(R), Copyright (c) 2008-2020, Alliance for Sustainable Energy, LLC.
```
Comment (TShapinsky) [inline alfalfa_worker/jobs/openstudio/lib/workflow/measures/alfalfa_prepare_idf/measure.rb:2]: This is not a real diff. I created this new measure and deleted old unsused code. Git somehow thought that is a move and a giant edit

**Review by anyaelena — DISMISSED** [review]
As discussed today.  Thanks Toby!

**Pull request merged**